### ISSUE TITLE
Atmos pipe and new rooms

### ISSUE THREAD

**Issue by Marflowerose** [tgstation/-tg-station#8465]
I was building an active filtration for air distro in the [singularity area](http://i.imgur.com/niuVrGK.png), left side. The volume pump that I have highlighted in red is the magical location where the air from air distro will not go. I had regular pipe there and tried volume pump, neither worked. The built area on the right is new room that I had assigned with blueprints. The left side is Engineering room and air goes normally in pipes of that area, but not in the new area. The newly designated area has powered APC that works and Air Alarm and I built the pipes after using blueprints.
In short, gas doesn't flow to new rooms with pipes.

**Comment by Miauw62** [comment]
this may explain some unreproduceable issues we've been seeing with pumps

**Comment by Menshin** [comment]
Unable to reproduce.
Just for information : did you tried unwrenching/wrenching back the bend red pipe at the right of the pump ?

I'm pretty sure this is caused by the last floating atmos bug which I have to find some time to tackle/fix. What happens is that parent pipenet of some pipes/atmos machinery is sometimes null (failed qdel suspected).

Every bit of information (and this picture is a good start) is welcome on this issue.

**Comment by Marflowerose** [comment]
I did not try to wrench the bend at the right of the pump but before I had the pump there, I had a pipe which did not get any pressure. Then I tried to add the pump and it did nothing.

**Comment by duncathan** [comment]
Out of curiosity, were those red pipes insulated pipes?

**Comment by Marflowerose** [comment]
There's insulated pipe before the blockade that worked IIRC, it was most likely about turfs/room assignations rather than insulated pipes.
